opencti-platform/opencti-graphql/tests/02-integration/04-manager/retentionManager-test.ts

```diff
@@ -224,14 +299,25 @@ describe('Retention Manager tests ', () => {
     expect(workbenchesToDelete.edges.length).toEqual(1); // workbench1 is the only workbench that has not been modified since 'before'
     expect(workbenchesToDelete.edges[0].node.id).toEqual(workbench1Id);
   });
+  it('should fetch the correct report to be deleted by a retention rule on knowledge', async () => {
+    // retention rule on workbenches not modified since 2023-07-01
+    const before = utcDate('2023-07-01T00:00:00.000Z');
+    const reportsToDelete = await getElementsToDelete(context, 'knowledge', before);
+    expect(reportsToDelete.edges.length).toEqual(1); // workbench1 is the only workbench that has not been modified since 'before'
+    expect(reportsToDelete.edges[0].node.id).toEqual(report1Id);
+  });
   it('should delete the fetched files and workbenches', async () => {
     // delete file
     await deleteElement(context, 'file', fileId); // should delete fileToTestRetentionRule
     const files = await allFilesForPaths(testContext, ADMIN_USER, [globalPath]);
-    expect(files.length).toEqual(8); // 7 files from index-file-test + the 2 created files - fileToTestRetentionRule that should have been deleted
+    expect(files.length).toEqual(1); // 7 files from index-file-test + the 2 created files - fileToTestRetentionRule that should have been deleted
```

Comment:
from 8 to 1 ? 
what about the comment

Reply:
oops, i changed it to test locally but forgot to revert it back, thanks!